gadgets/trace_tcp/ebpf/connect.h

```diff
@@ -53,36 +64,50 @@ static __always_inline int exit_tcp_connect(struct pt_regs *ctx, int ret,
 	__u64 pid_tgid = bpf_get_current_pid_tgid();
 	__u32 tid = pid_tgid;
 	struct tuple_key_t tuple = {};
-	struct gadget_process proc = {};
+	struct extended_info ei = {};
 	struct sock **skpp;
 	struct sock *sk;
+	__u32 *fd;
 
-	skpp = bpf_map_lookup_elem(&tcp_connect_sockets, &tid);
+	skpp = bpf_map_lookup_elem(&tcp_tid_sock, &tid);
 	if (!skpp)
 		return 0;
 
+	fd = bpf_map_lookup_elem(&tcp_tid_fd, &tid);
+	if (!fd)
+		return 0;
+	ei.fd = *fd;
+
 	if (ret)
```

Comment:
Not fully related to this PR, but can't we move the `if (ret)` to be the first one to avoid wasting cpu cycles on the map lookups in case ret != 0?